lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -201,7 +206,8 @@ def self.go_template_for(kind, name)
           %[(eq .involvedObject.kind "#{kind}")],
           %[(eq .involvedObject.name "#{name}")],
           '(ne .reason "Started")',
-          '(ne .reason "Created")'
+          '(ne .reason "Created")',
+          '(ne .reason "SuccessfulCreate")'
```

Comment:
You'd keep Scheduled, SuccessfulCreate, Pulling and Pulled, or just Scheduled? My concern here is generating so much noise that people stop reading the output. Keep in mind that the other events are always available in Splunk for more advanced debugging.